packages/client/src/app/pages/forms/list/archived/archived.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import type { FormWithStats } from '@ff-client/queries/forms';
+import translate from '@ff-client/utils/translations';
+
+import { ArchivedItem } from '../archived-item/archived-item';
+
+import { ArchivedItems, Button, Wrapper } from './archived.styles';
+
+type Props = {
+  data: FormWithStats[];
+};
+
+export const Archived: React.FC<Props> = ({ data }) => {
+  const [isVisible, setIsVisible] = useState(false);
+  const toggleVisibility = (): void => setIsVisible(!isVisible);
+
+  const archived = data && data.filter(({ archived }) => archived);
+  const isEmpty = (data && !archived.length) ?? true;
+
+  if (isEmpty) {
+    return;
```

Comment:
Please use `return null;` I have had issues with using `undefined` before and I think react officially asks to return `null` but I can't find it, so that might be bollocks.

```suggestion
    return null;
```